src/core/ContentTile.tsx

```diff
@@ -0,0 +1,167 @@
+              "flex justify-center max-h-[200px]",
+              !centerFeature && "pt-6 [&_img]:min-w-max [&_img]:h-[200px]",
+            )}
+          >
+            {feature}
+          </div>
+          {featureIcons && featureIcons?.length > 0 && (
+            <div className="absolute bottom-3 right-3 flex gap-1.5 bg-neutral-000 dark:bg-neutral-1300 rounded border border-neutral-200 dark:border-neutral-1100 px-2 py-1.5">
+              {featureIcons.map((icon, idx) => (
+                <Icon key={icon + idx} name={icon} size="18px" />
+              ))}
+            </div>
+          )}
+        </div>
+      );
+    }
+
+    if (featureType === "icon") {
+      return (
+        <div className="h-9">
+          {typeof feature === "string" ? (
+            <Icon name={feature as IconName} size="36px" />
+          ) : (
+            feature
+          )}
+        </div>
+      );
+    }
+
+    return null;
+  }, [centerFeature, cta, feature, featureIcons, featureType]);
+
+  return (
+    <div
+      className={cn("group/content-tile", cta && "cursor-pointer", className)}
+      {...(cta && {
+        onClick: handleClick,
+        onKeyDown: (e: React.KeyboardEvent) => {
+          if (e.key === "Enter" || e.key === " ") {
+            e.preventDefault();
+            handleClick();
+          }
+        },
+        tabIndex: 0,
+        role: "link",
+        "aria-label": title,
+      })}
+    >
+      {renderedFeature}
+      <div className="content-tile__content pr-4">
+        {title && (
+          <h2
+            className={cn(
+              "content-tile__title mt-4 mb-2 ui-text-h4 text-neutral-1300 dark:text-neutral-000",
+            )}
+          >
+            {title}
+          </h2>
+        )}
+        {description && (
+          <div className="content-tile__description mb-2 ui-text-p2 text-neutral-800 dark:text-neutral-500">
+            {description}
+          </div>
+        )}
+        {badges && badges.length > 0 && (
+          <div className="content-tile__badges mb-2 flex flex-wrap gap-2">
+            {badges.map((badge, idx) =>
+              typeof badge === "string" ? (
+                <Badge key={badge + idx} className="uppercase text-[10px]">
+                  {badge}
+                </Badge>
+              ) : (
+                <Badge
+                  key={badge.label + idx}
+                  className={cn("uppercase text-[10px]", badge.className)}
+                >
+                  {badge.label}
+                </Badge>
+              ),
+            )}
+          </div>
+        )}
+        {cta && (
+          <FeaturedLink
+            url="#"
+            additionalCSS={cn(
+              "pt-0 pointer-events-none font-medium items-center text-neutral-800 dark:text-neutral-500 group-hover/content-tile:text-neutral-1300 dark:group-hover/content-tile:text-neutral-000 transition-colors [&_svg]:group-hover/content-tile:left-0",
+            )}
+            iconColor="text-orange-600"
```

Comment:
in FlexibleTemplate  theu have the option to change the colors which include the FeatureLink color. are we able to pass a props to change it?

Reply:
TIL about custom modifiers